code-challenges.test.js

```diff
@@ -24,15 +24,45 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
-
+var fibonacciInputOne = 6
+var fibonacciInputTwo = 10
+
+describe("when 'fibonacciArray' is called", () => {
+  let fibonacciOutputOne = [1, 1, 2, 3, 5, 8]
+  let fibonacciOutputTwo = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+  it("returns an array with the amount of numbers requested in the Fibonacci sequence", () => {
+    expect(fibonacciArray(fibonacciInputOne)).toEqual(fibonacciOutputOne)
+    expect(fibonacciArray(fibonacciInputTwo)).toEqual(fibonacciOutputTwo)
+  })
+})
 
 // b) Create the function that makes the test pass.
-
-
-
-
+//make a function that takes a single number as an arguement (greater than 2) to determine length of array
+//create new array
+//map will then return value + value to each iteration
+
+// const fibonacciArray = (i) => {
+//   let newFibArray = []
+//   return newFibArray.forEach((item, i) => {
+//     (item + item++)
+//   });
+//   return newFibArray
+// }
+
+//^could not use map or foreach -- while I am sure there is a way....could not get it to work.  Decided to go for a for loop instead
```

Comment:
awesome pseudocode

Reply:
for using `.map()` for this problem, you'd need to initialize the array with something like `Array(10).fill(0)` and then map over this array